core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -31,6 +33,9 @@ class ApplicationInfo private[spark](
     val memoryPerExecutorMB: Option[Int],
     val attempts: Seq[ApplicationAttemptInfo])
 
+@JsonIgnoreProperties(
+  value = Array("startTimeEpoch", "endTimeEpoch", "lastUpdatedEpoch"),
```

Comment:
No, this just avoids trying to deserialize them, which would cause an error because these properties have no setter.